pkgs/development/compilers/emscripten/default.nix

```diff
@@ -42,17 +42,7 @@ stdenv.mkDerivation rec {
   patches = [
     (substituteAll {
       src = ./0001-emulate-clang-sysroot-include-logic.patch;
-      resourceDir = "${llvmEnv}/lib/clang/${llvmPackages.release_version}/";
-    })
-    # https://github.com/emscripten-core/emscripten/pull/18219
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
@SuperSandro2000 sorry for missing! adjusted with 0761348.